docs/conversion_examples_gallery/recording/spikeglx.rst

```diff
@@ -7,7 +7,41 @@ Install NeuroConv with the additional dependencies necessary for reading SpikeGL
 
     pip install neuroconv[spikeglx]
 
-Convert ap or lf SpikeGLX streams to NWB using
+
+
+SpikeGLXConverter
+~~~~~~~~~~~~~~~~~
+
+We can easily convert all data stored in the native SpikeGLX folder structure to NWB using
+:py:class:`~neuroconv.converters.SpikeGLXConverterPipe`.
+
+.. code-block:: python
+
+    >>> from datetime import datetime
+    >>> from dateutil import tz
+    >>> from pathlib import Path
+    >>> from neuroconv.converters import SpikeGLXConverterPipe
+    >>>
+    >>> folder_path = f"{ECEPHY_DATA_PATH}/spikeglx/Noise4Sam_g0"
+    >>> converter = SpikeGLXConverterPipe(folder_path=folder_path)
```

Comment:
Allowing verbose here because it's cleaner to handle with ELLIPSIS as an output field in the last step